src/test/unit/objects/test_virtual_file_path.py

```diff
@@ -2,17 +2,25 @@
 from test.common_helper import create_test_file_object, create_test_firmware
 
 
-def test_add_file_object():
+def test_root():
     root = create_test_firmware()
     assert root.virtual_file_path[root.uid] == [root.uid], 'virtual file path of root file not correct'
+
+
+def test_add_file_object_to_firmware():
+    root = create_test_firmware()
     child_one = create_test_file_object()
```

Comment:
To be super picky, I'd comment that the distinction of *one* isn't necessary anymore. Same for the test below where `grandchild` would be the way to go. #nitpicking